### DISCUSSION TITLE
Fail2ban and IPV6

### DISCUSSION THREAD

**Discussion by danardf** [fail2ban/fail2ban#3161] [General]
Hi everyone.

I'd like to know what's the min version to handle IPV6? 
I'm currently using 0.8.14 and I don't think this is the good version to handle IPV6.
Maybe I should use 0.11.2, isn't it?

**Comment by sebres** [comment]
v.0.10 is the minimal version supporting IPv6

**Reply by danardf** [reply]
Ok thanks 
Good to know.

Have a nice day :)
